stores-api/governance-api/src/main/java/com/bloxbean/cardano/yaci/store/api/governance/controller/LocalDRepDistrController.java

```diff
@@ -15,12 +15,12 @@
 import org.springframework.web.server.ResponseStatusException;
 
 @RestController
-@RequestMapping("${apiPrefix}/local/dRep")
+@RequestMapping("${apiPrefix}/governance/live/dRep")
```

Comment:
@Sotatek-HuyLe3a Can you please change dRep to lower case "drep"?

I think we should change it to plural "dreps" to align with other controllers like "blocks", "scripts" etc.

Reply:
i fixed this